elixir/markdown/lib/markdown.ex

```diff
@@ -11,37 +11,53 @@ defmodule Markdown do
     "<h1>Header!</h1><ul><li><em>Bold Item</em></li><li><i>Italic Item</i></li></ul>"
   """
 
-  #This should probably be a pipeline instead of all those nested functions
+  #Refactored into a pipeline to get rid of the nested functions
   @spec parse(String.t()) :: String.t()
   def parse(m) do
-    patch(Enum.join(Enum.map(String.split(m, "\n"), fn t -> process(t) end)))
-  end
+    m
```

Comment:
Is there a better variable that could be used rather than `m`?

Reply:
No response to this question yet

Reply:
I have changed it to "markdown_text". I'm not 100% sure if this is a good choice. Maybe 'source_text'?

Reply:
I think either sounds good, maybe `markdown_text` could be better as it is a bit more specific?